src/website/src/pages/ec/utilities/shadow/docs/usage.mdx

```diff
@@ -17,24 +17,6 @@ Available values are:
 
 - ecl-u-shadow-none (remove all shadow)
 - ecl-u-shadow-1 (elevation 1)
-- ecl-u-shadow-2 (elevation 2)
-- ecl-u-shadow-3 (elevation 3)
-
-## Inner shadows
-
-By using classes `ecl-u-shadow-inner-*` you can manage inner shadow of an element.
-Available values are:
-
-- ecl-u-shadow-inner-1 (depth 1)
-- ecl-u-shadow-inner-2 (depth 2)
-
-## Negative shadows
-
-By using classes `ecl-u-shadow-negative-*` and `ecl-u-shadow-negative-inner-*` you can manage negative shadow of an element (for dark background).
-Available values are:
-
-- ecl-u-shadow-negative-1 (elevation 1)
-- ecl-u-shadow-negative-2 (elevation 2)
-- ecl-u-shadow-negative-3 (elevation 3)
-- ecl-u-shadow-negative-inner-1 (depth 1)
-- ecl-u-shadow-negative-inner-2 (depth 2)
+- ecl-u-shadow-6 (elevation 6)
```

Comment:
this is a personal consideration, i don't understand this "scale", why is not simply 1-2-3-4, it feels a bit randomish to pull an existing level of shadow like this..

Reply:
I think we discussed this already, but that's how the shadows are defined at design level: the "elevation" is kind of the z-index for them. And they wanted to keep the possibility to add other shadows in-between
![image](https://github.com/ec-europa/europa-component-library/assets/25579221/539d23a5-4121-4272-bfc2-76643b5fccbd)
Now, we could use 1, 2, 3, 4 for our utilities, but then they won't be in sync with the terminolgy they use, or with the sass maps